client/my-sites/plan-features/actions.jsx

```diff
@@ -21,33 +21,43 @@ import { getPlanClass, isMonthly } from 'lib/plans/constants';
 import { recordTracksEvent } from 'state/analytics/actions';
 
 const PlanFeaturesActions = ( {
+	available = true,
 	canPurchase,
 	className,
-	available = true,
+	currentSitePlan,
 	current = false,
-	primaryUpgrade = false,
 	freePlan = false,
-	onUpgradeClick = noop,
+	manageHref,
+	isLandingPage,
 	isPlaceholder = false,
 	isPopular,
 	isInSignup,
-	translate,
-	manageHref,
-	isLandingPage,
+	onUpgradeClick = noop,
 	planName,
-	currentSitePlan,
 	planType,
+	primaryUpgrade = false,
+	selectedPlan,
 	recordTracksEvent: trackTracksEvent,
+	translate,
 } ) => {
 	let upgradeButton;
-	const classes = classNames(
-		'plan-features__actions-button',
-		{
-			'is-current': current,
-			'is-primary': ( primaryUpgrade && ! isPlaceholder ) || isPopular,
-		},
-		className
-	);
+	const classes = selectedPlan
+		? classNames(
+				'plan-features__actions-button',
+				{
+					'is-current': current,
+					'is-primary': planName === selectedPlan,
```

Comment:
It's a pretty bad idea to compare by the name of the plan because:

* we rely on capitalization of the plan name, and since it arrives from a parameter, we should be case-agnostic
* at some point we might end up localizing the plan names

At this point you can use `planType`, which will contain the plan slug (e.g. `jetpack_premium` or `personal-bundle`) - you can refer to the comments above for more information.

Reply:
point taken 👍